c2corg_ui/templates/helpers/view.html

```diff
@@ -45,7 +19,7 @@
     </div>
     <span class="detail-text accordion">
       <article class="value">
-        % if personal:
+        % if is_personal(doc):
```

Comment:
You have already run ``is_personal`` a few lines above.
I would recommend to use variables to store the results once for all at the beginning of the function:
```
personal = is_personal(doc)
collaborative = is_collaborative(doc) if not personal else False
```